### ISSUE TITLE
Need to find/replace eCPIC references with Folio 

### ISSUE THREAD

**Issue by AndreaSigz** [GSA/digitalgov.gov#197]
Per Sarah Crane's email on 11/6, eCPIC changed its name to Folio, so we should update the name change on the site @jeremyzilar 

https://gsablogs.gsa.gov/folio/2017/11/03/folio-evolution/

**Comment by jeremyzilar** [comment]
Are we replacing all references to `https://www.ecpic.gov/` with `https://gsablogs.gsa.gov/folio/`?
And changing all references of `eCPIC` with `Folio`?

**Comment by jeremyzilar** [comment]
I propose that we remove the links to eCPIC on the services navigation and redirect the main eCPIC services page https://www.digitalgov.gov/services/electronic-capital-planning-and-investment-control-ecpic/ to their new site.

**Comment by AndreaSigz** [comment]
For the links you remove, are you going to put a redirect in, because if ppl have bookmarked those links, they will break.
And according to DAP, there were 49 unique page views to the main eCPIC page in the last 30 days

**Comment by ToniBonittoGSA** [comment]
That post is confusing... They mention both "eCPIC application" and "eCPIC program" -- are they changing the name of the eCPIC application to Folio, or will the name of their overall _program_ now be called Folio (but still offer a tool named eCPIC)? 
The blog footer has, "Copyright eCPIC Folio. All Rights Reserved."

Andrea, did you mean change _eCPIC_ to _Folio_ on this (and other?) pages https://www.digitalgov.gov/services/electronic-capital-planning-and-investment-control-ecpic/ ?

~ also, redirecting would "break" consistency of our Services info; all go to an info page

**Comment by AndreaSigz** [comment]
Yes, they changed their program name to Folio. Not sure about the application name.

Toni, yep, we should update the program name on all their pages.

If @jeremyzilar was talking about deleting the pages all together, we should have a redirect. If he is talking about just removing them from the Services menu, that's different
